pkg/cmd/repo/deploy-key/deploy-key.go

```diff
@@ -0,0 +1,20 @@
+package deploykey
+
+import (
+	cmdAdd "github.com/cli/cli/v2/pkg/cmd/repo/deploy-key/add"
+	cmdDelete "github.com/cli/cli/v2/pkg/cmd/repo/deploy-key/delete"
+	cmdList "github.com/cli/cli/v2/pkg/cmd/repo/deploy-key/list"
+	"github.com/cli/cli/v2/pkg/cmdutil"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdDeployKeys(f *cmdutil.Factory) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "deploy-key <command>",
+		Short: "Deploy a new ssh key to current repository",
```

Comment:
To avoid confusion, let's stick to always saying "deploy key" as a noun and avoid saying "deploy" as a verb while we're in the same context.
```suggestion
		Short: "Manage deploy keys for the repository",
```